config/config.exs

```diff
@@ -15,5 +16,27 @@ config :lambda_ethereum_consensus, BeaconApi.Endpoint,
     layout: false
   ]
 
+# Configures peer discovery
```

Comment:
These are the ones corresponding to Sepolia, right? If that's the case, maybe a comment would help?

Reply:
Those are for mainnet, and it's commented on the bootnodes option.